### ISSUE TITLE
Add GO-SHIP line to microbial page

### ISSUE THREAD

**Issue by clairedavies** [PlanktonTeam/BioOceanObserver#201]
contour plots against latitude

**Comment by jaseeverett** [comment]
I think this is done @clairedavies ? Can this be closed now?

**Comment by clairedavies** [comment]
No, this is still just at the development stage, a quick fix to satisfy ARDC. The function is not generic and is still in BOO. I was waiting for the AM to add some project info to the dataset so that I could improve the function before transferring to planktonr.

**Comment by jaseeverett** [comment]
Ok great.
